Sources/PayPalDataCollector/PayPalDataCollectorEnvironment.swift

```diff
@@ -0,0 +1,16 @@
+import PPRiskMagnes
+
+/// Enum of environments to use with PayPalDataCollector
+public enum PayPalDataCollectorEnvironment {
```

Comment:
Curious why we didn't make it `PayPalDataCollector.Environment` (place it inside `PayPalDataCollector` type).

Reply:
@vladfilyakov just following the convention in this SDK, where every enum is in a diferent file.